src/compiler/emitter.ts

```diff
@@ -3685,7 +3685,11 @@ export function createPrinter(printerOptions: PrinterOptions = {}, handlers: Pri
     }
 
     function emitImportClause(node: ImportClause) {
-        if (node.isTypeOnly) {
+        if (node.phaseModifier !== undefined) {
+            emitTokenWithComment(node.phaseModifier, node.pos, writeKeyword, node);
+            writeSpace();
+        }
+        else if (node.isTypeOnly) {

```

Comment:
I have this in case somebody passes in an object from an old TS version. Is this relevant for TS' API contract, or would removing it not be considered a breaking change?

Reply:
We don't support mixing TS versions between a given `Node` and the emitter, so that shouldn't be a concern. Inside the compiler we should avoid depending on deprecated functionality.